src/components/AddCompanies/AddCompanies.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, FormGroup, ControlLabel, FormControl, Col } from 'react-bootstrap';
+import './styles.less';
+
+const propTypes = {
+    addCompany: PropTypes.func.isRequired,
+};
+
+class AddCompanies extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			name: '',
+			address: '',
+			phone: '',
+			revenue: '',
+		};
```

Comment:
Here I set up state, this will manage the values in the Bootstrap Form components... There may be a better Bootstrappier way to do this... I'm just not familiar with Bootstrap so I am leaning on what I know will work.